### ISSUE TITLE
Targeting S+ (version 31 and above) requires that one of FLAG_IMMUTABLE or FLAG_MUTABLE be specified when creating a PendingIntent

### ISSUE THREAD

**Issue by brianjcoan** [muzei/muzei#789]
Seeing this stacktrace:

```
E  Writing exception to parcel (Ask Studio Bot)
   java.lang.IllegalArgumentException: myapp: Targeting S+ (version 31 and above) requires that one of FLAG_IMMUTABLE or FLAG_MUTABLE be specified when creating a PendingIntent.
   Strongly consider using FLAG_IMMUTABLE, only use FLAG_MUTABLE if some functionality depends on the PendingIntent being mutable, e.g. if it needs to be used with inline replies or bubbles.
   	at android.app.PendingIntent.checkFlags(PendingIntent.java:401)
   	at android.app.PendingIntent.getActivityAsUser(PendingIntent.java:484)
   	at android.app.PendingIntent.getActivity(PendingIntent.java:470)
   	at android.app.PendingIntent.getActivity(PendingIntent.java:434)
   	at com.google.android.apps.muzei.api.provider.MuzeiArtProvider.getArtworkInfo(MuzeiArtProvider.java:663)
   	at com.google.android.apps.muzei.api.provider.MuzeiArtProvider.call(MuzeiArtProvider.java:531)
   	at android.content.ContentProvider.call(ContentProvider.java:2511)
   	at android.content.ContentProvider$Transport.call(ContentProvider.java:525)
   	at android.content.ContentProviderNative.onTransact(ContentProviderNative.java:295)
   	at android.os.Binder.execTransactInternal(Binder.java:1285)
   	at android.os.Binder.execTransact(Binder.java:1244)
I  Provider for content://myapp.muzei.provider/1949 crashed while opening artwork info (Ask Studio Bot)
   android.os.RemoteException: myapp: Targeting S+ (version 31 and above) requires that one of FLAG_IMMUTABLE or FLAG_MUTABLE be specified when creating a PendingIntent.
   Strongly consider using FLAG_IMMUTABLE, only use FLAG_MUTABLE if some functionality depends on the PendingIntent being mutable, e.g. if it needs to be used with inline replies or bubbles.
   	at com.google.android.apps.muzei.util.ContentProviderClientCompat$call$2.invokeSuspend(ContentProviderClientCompat.kt:58)
   	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
   	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
   	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:571)
   	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:750)
   	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:678)
   	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:665)

```
guessing it should be an easy fix in `MuzeiArtProvider.getArtworkInfo`

**Comment by ianhanniballake** [comment]
It has already been fixed in #747. Upgrade to [Muzei API 3.4.1](https://github.com/muzei/muzei/releases/tag/api3.4.1).

**Comment by brianjcoan** [comment]
Thanks.  Apologies for not catching that.  Usually Studio lets me know when dependencies are out of date, but didn't flag muzei.
